industrial_robot_client/package.xml

```diff
@@ -34,4 +34,6 @@
   <run_depend>urdf</run_depend>
   <run_depend>industrial_msgs</run_depend>
   <run_depend>industrial_utils</run_depend>
+  <run_depend>rviz</run_depend>
```

Comment:
Something to consider: in light of ros-industrial/ros_industrial_issues#50, we might not want to add `rviz` here (as that would make RViz again a dependency of all robot drivers that depend on `industrial_robot_client`).